packages/conf-pkg-config/conf-pkg-config.1.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "1.2"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "http://www.freedesktop.org/wiki/Software/pkg-config/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "git://github.com/ocaml/opam-repository"
+license: "GPL"
+build: [
+  ["pkg-config" "--help"]
+]
+install: [
+  ["ln" "-s" "/usr/local/bin/pkgconf" "%{bin}%/pkg-config" {os = "openbsd"}]
```

Comment:
shouldn't the `{ os = "openbsd" }` be outside of the `[ ... ]` build instructions?

Reply:
The manual seems to suggest that either way is fine: 

> `install: [ [ <string> { <filter> } ... ] { <filter> } ... ]`: the list of commands that will be run in order to install the package. [...]

and `opam lint` does not complain, so I guess it should be fine?

Reply:
according to a `git grep`, both are used in opam-repository, so this is fine with me.

Reply:
I agree with @hannesm that should be outside. Otherwise only `"%{bin}%/pkg-config"` will be present on openbsd but the rest will be present on any plateform.

The CI doesn't fail because it seems that `ln -s foo` is a valid command.

Reply:
@kit-ty-kate raises a good point here.